IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/NewsArticleViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  NewsArticleViewModel.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+final class NewsArticleViewModel {
+    
+    let imageUrl: String
```

Comment:
I thought the same when I first saw this ViewModel, but usually you only define observable properties for values you know will change through object's life-cycle.

In this particular case, none of this values will change after the object is instantiated, they will never trigger a change notification, so making these properties observable is not strictly required (unless the frameworks we are using requires that, but that would be odd. They should support a classic one-time binding).